lms/djangoapps/lms_xblock/mixin.py

```diff
@@ -142,6 +144,62 @@ def _get_user_partition(self, user_partition_id):
 
         raise NoSuchUserPartitionError("could not find a UserPartition with ID [{}]".format(user_partition_id))
 
+    def get_parent_unit(self):
+        """
+        Finds xblock's parent unit if it exists.
+
+        Returns:
+            xblock: Returns the parent unit xblock if it exists.
+            If no parent unit exists, returns None.
+        """
+        xblock = self
+        while xblock:
+            xblock = xblock.get_parent()
```

Comment:
Definitely will move the new version there and share the code.  Currently, that method is being tested in edxnotes/tests.py with all of the other edxnotes helper functions.  Should I make a new test file and duplicate the set up code from there to test this one specific function in the new location, or is it ok to leave the test in edxnotes and leave a note in the docstring about where the actual implementation is?